memory-leak-tests/src/test/scala/zio/stream/ZStreamMemoryLeakSpec.scala

```diff
@@ -0,0 +1,31 @@
+package zio.stream
+
+import zio._
+
+class ZStreamMemoryLeakSpec extends MemoryLeakSpec {
```

Comment:
If their execution time is only 10 seconds each, I think we can take it and put these with all the other stream tests, but run them only on the JVM and potentially only during CI (we can use an envvar for that). The total CI time won't get worse because the JVM suites finish a few minutes earlier than the JS/Native ones